### PR TITLE
feat: add new tooltip tables for easier setting of general MSU tooltips

### PR DESCRIPTION
**Pull request by TaroEld**

### CODE REVIEW

File: ui/mods/msu/mod_settings/settings_screen.js [hunk 1]
```diff
@@ -98,6 +98,12 @@ ModSettingsScreen.prototype.createDIV = function (_parentDiv)
 	this.mModPageScrollContainer = this.mModPageContainer.findListScrollContainer();
 };
 
+ModSettingsScreen.prototype.bindTooltips = function ()
+{
+	this.mDialogContainer.find('.main-cancel-button').bindTooltip({ contentType: 'ui-element', elementId: MSU.TooltipIdentifiers.ModSettings.Main.Cancel});
+	this.mDialogContainer.find('.main-save-button').bindTooltip({ contentType: 'ui-element', elementId: MSU.TooltipIdentifiers.ModSettings.Main.OK});
+};
+
```
Comment (Enduriel) [inline ui/mods/msu/mod_settings/settings_screen.js:106]: Why not just create variables for these?

File: ui/mods/msu/mod_settings/settings_screen.js [hunk 2]
```diff
@@ -118,6 +124,8 @@ ModSettingsScreen.prototype.unbindTooltips = function ()
 	{
 		setting.unbindTooltip();
 	});
+	this.mDialogContainer.find('.main-cancel-button').unbindTooltip();
+	this.mDialogContainer.find('.main-save-button').unbindTooltip();
```
Comment (Enduriel) [inline ui/mods/msu/mod_settings/settings_screen.js:128]: As above

File: ui/mods/msu/tooltip_identifiers.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        keyCache.pop();
+  	    return false;
+    }
+    keyCache.forEach(function(_cachedKey){
+        _value += _cachedKey + ".";
+    })
+    curObj[_key] = _value.slice(0, -1);
+    keyCache.pop()
+    if (idx == Object.keys(curObj).length) keyCache.pop();
+}
```
Comment (Enduriel) [inline ui/mods/msu/tooltip_identifiers.js:45]: Should not be globally namespaces, put it in the MSU table

**Review by Enduriel — CHANGES_REQUESTED** [review]

File: ui/mods/msu/tooltip_identifiers.js (hunk 3 above)
Comment (TaroEld) [reply ui/mods/msu/tooltip_identifiers.js:45]: Variable was deleted instead for the time being

File: ui/mods/msu/mod_settings/settings_screen.js (hunk 1 above)
Comment (TaroEld) [reply ui/mods/msu/mod_settings/settings_screen.js:106]: Fine for now

File: ui/mods/msu/mod_settings/settings_screen.js (hunk 2 above)
Comment (TaroEld) [reply ui/mods/msu/mod_settings/settings_screen.js:128]: Fine for now

**Pull request merged**